components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -82,6 +83,21 @@ def update_job_metadata_and_tags(db_cursor, job_id, table_prefix, tag_ids, archi
     )
 
 
+def update_archive_metadata(db_cursor, archive_stats):
+    archive_stats_defaults = {
+        "begin_timestamp": 0,
+        "end_timestamp": 0,
+        "creator_id": "",
+        "creation_ix": 0,
+    }
+    for k, v in archive_stats_defaults.items():
+        archive_stats.setdefault(k, v)
+    keys = ", ".join(archive_stats.keys())
+    value_placeholders = ", ".join(["%s"] * len(archive_stats))
+    query = f"INSERT INTO {ARCHIVES_TABLE_NAME} ({keys}) VALUES ({value_placeholders})"
```

Comment:
The archives table name is technically `<table_prefix>archives`. Although the user has no way of setting the table prefix right now, it is an option that's part of the clp metadata [connection parameters](https://github.com/y-scope/clp/blob/acefdcd80581623e32ee9a11deb6db90b8765359/components/clp-py-utils/clp_py_utils/clp_config.py#L146). So to avoid getting out of sync with the rest of the codebase, we should continue to pull the table prefix from the connection parameters like `update_tags` does. What we could use a constant for is the string "archives".